.pre-commit-config.yaml

```diff
@@ -94,12 +94,3 @@ repos:
     additional_dependencies:
     - Sphinx>=3.1.2
     pass_filenames: false
-
-- repo: local
-  hooks:
-  - id: pylint
-    language: system
-    name: PyLint
-    files: \.py$
-    entry: python -m pylint
-    args: []
```

Comment:
I've decided that it's cleaner to use `stages: [manual]` here. I believe it will achieve what you intended in a bit cleaner fashion. I'll make the change.